### ISSUE TITLE
[naming-convention] Configuration invalid (unexpected error) when using config all

### ISSUE THREAD

**Issue by fernandopasik** [typescript-eslint/typescript-eslint#1472]
When using the config `all` the config applied for this rule is

```
"@typescript-eslint/naming-convention": "error",
```
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/all.json#L24

**Repro**

```JSON
{
  "extends": [
    "plugin:@typescript-eslint/all"
  ]
}
```

**Expected Result**

Run normally.

**Actual Result**

Throws an unexpected error

```
Error: .eslintrc.json » plugin:@typescript-eslint/all:
        Configuration for rule "@typescript-eslint/naming-convention" is invalid:
        Value [] should NOT have fewer than 1 items.
```

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.16.0` |
| `@typescript-eslint/parser`        | `2.16.0` |
| `TypeScript`                       | `3.7.5` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `12.13.1` |

**Comment by fernandopasik** [comment]
Trying to disable the rule in `.eslintrc` with 

```
@typescript-eslint/naming-convention": "off"
```

still does not make the error go away

**Comment by bradzacher** [comment]
Fixed in #1455

**Comment by fernandopasik** [comment]
Thanks @bradzacher
